lib/dry/transaction/step.rb

```diff
@@ -10,9 +10,13 @@ class Step
       UNDEFINED = Object.new.freeze
       RETURN = -> x { x }
 
-      include Wisper::Publisher
+      include Dry::Events::Publisher[object_id]
```

Comment:
`object_id` is definitely a stable publisher identifier but I wonder if we can avoid using it in the times when we have a more meaningful name? How about `include Dry::Events::Publisher[name || object_id]`?

Reply:
Part of my thinking of avoiding using object_id is because at some point accessing the object_id may prevent the runtime for GC'ing efficiently (this is something tenderlove has mentioned in some talks of his). So I figure if we can avoid it, why not?

Reply:
@timriley I will update the code today!